src/oprf.ts

```diff
@@ -12,7 +12,6 @@ import {
     type GroupID,
     type Scalar
 } from './groupTypes.js'
-
```

Comment:
lint enforcement seems to be insufficient in CI

Reply:
linter does not check for code formatting.

Reply:
updated the configuration now